source/2019-12-06-the-ember-times-issue-127.md

```diff
@@ -75,13 +77,19 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Become an Ember Author for the Countdown to The New Year Blog Series ✍️🎄](https://discuss.emberjs.com/t/writers-wanted-countdown-to-the-new-year-blog-series/17273)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+This year the Ember Blog will feature **one** fun, useful and interesting **Ember addon each day**. And we are still looking for writers to make the official **Countdown to the New Year** blog post series a reality!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+If you are interested in contributing to the series, you need first experience with Markdown and Git to get started - that's it! If you would like to write a short article for the Ember blog in _DecEmber_ yourself, feel free to [sign up for any of the issues with the DecEmber label](https://github.com/ember-learn/ember-blog/labels/decEmber) and take a look [at our contribution guide](https://github.com/ember-learn/ember-blog/blob/master/source/december-2019-blog-series-template.md).
```

Comment:
I thought it was a good idea to mention that all one needs is some experience with Markdown and Git. I think we could even mention whom one can ask for help with making a PR to the repo (e.g. `dev-ember-learning` or a couple of specific people by their names).

Reply:
Yes, that sounds like a good idea! I can update that accordingly